File: 1528-kids-with-the-greatest-number-of-candies/kids-with-the-greatest-number-of-candies.py
class Solution(object):
    def kidsWithCandies(self, candies, extraCandies):
        output = []
        maxCandies = max(candies)
        for i in range(len(candies)):
            if candies[i] + extraCandies >= maxCandies:
                output.append(True)
            else:
                output.append(False)
            
        return output

File: 1528-kids-with-the-greatest-number-of-candies/kids-with-the-greatest-number-of-candies.ts
function kidsWithCandies(candies: number[], extraCandies: number): boolean[] {
 let maxCandies: number =candies[0];
 let output: boolean[] = [];
 for(let i=0;i<=candies.length-1;i++){  
    if (candies[i]> maxCandies){
        maxCandies= candies[i];
    }
 }
 for(let i=0; i<=candies.length-1;i++){
    if(candies[i]+ extraCandies>= maxCandies){
        output.push(true);
    }
    else{
        output.push(false);
    }
 }
 return output;
};